apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -96,7 +103,7 @@ def count(self) -> int:
         """
         Return the number of rows in the dataframe. Same as `len(df)`.
         """
-        # A.domain.shape at the tiledb level gives us the 0..2^63 range which is not what we want
+        # A.domain.shape at the tiledb level gives us the 0..2^63-1 range which is not what we want
```

Comment:
not sure I understand this comment - it may be obsolete.  How does `shape` or domain relate to count/nnz?

Reply:
@bkmartinjr stale indeed